docs/tools/scripts.md

```diff
@@ -25,4 +25,4 @@ These scripts assume you are using bash.
 
 1. **createsuperuser.sh** - creates a default superuser.
 
-    1. This assumes that `DJANGO_SUPERUSER_USERNAME` and `DJANGO_SUPERUSER_PASSWORD` are set in `.env.dev`
+    1. This assumes that `DJANGO_SUPERUSER_USERNAME` and `DJANGO_SUPERUSER_PASSWORD` are set in `.env.docker`
```

Comment:
So it's `.env.docker` and `.env.local`. Eventually we could have a `.env.production`, which also uses docker. We'll think about that when we get there.